internal/system/agent/services.go

```diff
@@ -0,0 +1,44 @@
+package agent
+
+import (
+	"fmt"
+)
+
+const (
+	START   = "start"
```

Comment:
Using capitals for const is not Go idiomatic.

Reply:
Additionally - do you need string consts? This is handled more elegantly with [iota](https://splice.com/blog/iota-elegant-constants-golang/)